test/unit/callbacks_and_delays_test.lua

```diff
@@ -105,5 +106,43 @@ function g.test_delays_and_scan_callbacks(cg)
     t.assert(check_full_scan_delay)
     t.assert(check_iteration_delay)
     t.assert_equals(full_scan_success_counter, 2)
-    t.assert_equals(space:count{}, 0)
+    t.assert_equals(space:count(), 0)
+end
+function g.test_error_callback(cg)
```

Comment:
There were 2 empty lines between tests and now 0. I would like 1 =)
```suggestion
end

function g.test_error_callback(cg)
```

Reply:
Sorry, I didn't check what was changed by code suggestion

Reply:
Fixed